flink-table-store-hive/flink-table-store-hive-catalog/pom.xml

```diff
@@ -445,6 +445,28 @@ under the License.
                     <groupId>org.pentaho</groupId>
                     <artifactId>pentaho-aggdesigner-algorithm</artifactId>
                 </exclusion>
+                <!-- Exclude the older hadoop-common-->
+                <exclusion>
+                    <groupId>org.apache.hadoop</groupId>
+                    <artifactId>hadoop-common</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
```

Comment:
What causes the change here?

Reply:
Because the Hive E2E test failed before, I debugged and found that `Configuration#getPropsWithPrefix` is introduced since hadoop-common 2.8, while
`mvn dependency:tree -Dincludes=org.apache.hadoop:hadoop-common`
reveals we depended on hadoop-common 2.7.2
```
[INFO] org.apache.flink:flink-table-store-hive-catalog:jar:0.3-SNAPSHOT
[INFO] \- org.apache.hive.hcatalog:hive-webhcat-java-client:jar:2.3.9:test
[INFO]    \- org.apache.hadoop:hadoop-common:jar:2.7.2:compile
```

So I changed it in the first place. But later, I realized that it'd be better not to call `Configuration#getPropsWithPrefix` but forgot to revert this back.

I'll revert it.